### PR TITLE
Update CHANGELOG and VERSION for v0.5.0 release

### PR DESCRIPTION
**Pull request by elton-furtado**
Set VERSION to v0.5.0
Update CHANGELOG for 0.5.0 release

Signed-off-by: Elton Furtado <efurtado@vmware.com>

### CODE REVIEW

File: CHANGELOG/CHANGELOG-0.5.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Changelog 0.5
+
+## 0.5.0 - 2023-03-17
+
+### Added
+
+- Resync Theia Manager at startup. ([#150](https://github.com/antrea-io/theia/pull/150), [@yanjunz97])
```
Comment (jianjuns) [inline CHANGELOG/CHANGELOG-0.5.md:7]: Can we use the same tense for all sentences?

File: CHANGELOG/CHANGELOG-0.5.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  * Supports removing stale Spark jobs and database entries after restart.
```
Comment (jianjuns) [inline CHANGELOG/CHANGELOG-0.5.md:8]: Support

File: CHANGELOG/CHANGELOG-0.5.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  * Adding back queued Network Policy Recommendations job to sync list.
+- Service Dependency Graph. ([#142](https://github.com/antrea-io/theia/pull/142), [@Dhruv-J])
+  * Introduce sankey graph for visualizing traffic between pods and services.
```
Comment (jianjuns) [inline CHANGELOG/CHANGELOG-0.5.md:11]: Pods and Services

We capitalize K8s resource and CRD kinds.

File: CHANGELOG/CHANGELOG-0.5.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Changelog 0.5
+
+## 0.5.0 - 2023-03-17
+
+### Added
+
+- Added Theia Manager resync at startup. ([#150](https://github.com/antrea-io/theia/pull/150), [@yanjunz97])
+  * Support removing stale Spark jobs and database entries after restarting Theia Manager.
+  * Added backed-up Network Policy Recommendations jobs to the sync list.
+- Service Dependency Graph. ([#142](https://github.com/antrea-io/theia/pull/142), [@Dhruv-J])
```
Comment (jianjuns) [inline CHANGELOG/CHANGELOG-0.5.md:10]: Added?

**Review by jianjuns — COMMENTED** [review]
A nit.

File: CHANGELOG/CHANGELOG-0.5.md (hunk 4 above)
Comment (jianjuns) [reply CHANGELOG/CHANGELOG-0.5.md:10]: You forgot to update this one?

File: CHANGELOG/CHANGELOG-0.5.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+## 0.5.0 - 2023-03-17
+
+### Added
+
+- Added Theia Manager resync at startup. ([#150](https://github.com/antrea-io/theia/pull/150), [@yanjunz97])
+  * Support removing stale Spark jobs and database entries after restarting Theia Manager.
+  * Scheduled Network Policy Recommendations jobs are added back to the periodical sync list.
+- Service Dependency Graph. ([#142](https://github.com/antrea-io/theia/pull/142), [@Dhruv-J])
+  * Introduced a Sankey graph for visualizing traffic between Pods and Services.
```
Comment (salv-orlando) [inline CHANGELOG/CHANGELOG-0.5.md:11]: Is it a Sankey diagram? Would it be better to describe it as a "directed graph"?

File: CHANGELOG/CHANGELOG-0.5.md (hunk 4 above)
Comment (elton-furtado) [reply CHANGELOG/CHANGELOG-0.5.md:10]: "Scheduled Network Policy Recommendations jobs are added back to the periodical sync list"

File: CHANGELOG/CHANGELOG-0.5.md (hunk 5 above)
Comment (elton-furtado) [reply CHANGELOG/CHANGELOG-0.5.md]: changed to Sankey diagram

File: CHANGELOG/CHANGELOG-0.5.md (hunk 4 above)
Comment (jianjuns) [reply CHANGELOG/CHANGELOG-0.5.md:10]: I meant should we say: "Added Service Dependency Graph" to be consistent with other items.

File: CHANGELOG/CHANGELOG-0.5.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+## 0.5.0 - 2023-03-17
+
+### Added
+
+- Added Theia Manager resync at startup. ([#150](https://github.com/antrea-io/theia/pull/150), [@yanjunz97])
+  * Support removing stale Spark jobs and database entries after restarting Theia Manager.
+  * Scheduled Network Policy Recommendations jobs are added back to the periodical sync list.
+- Service Dependency Graph. ([#142](https://github.com/antrea-io/theia/pull/142), [@Dhruv-J])
+  * Introduced a Sankey diagram for visualizing traffic between Pods and Services.
```
Comment (salv-orlando) [inline CHANGELOG/CHANGELOG-0.5.md:11]: Is it a Sankey diagram? Maybe should we consider it a "directed graph"?

Comment (elton-furtado) [reply CHANGELOG/CHANGELOG-0.5.md]: changed to directed graph

File: CHANGELOG/CHANGELOG-0.5.md (hunk 4 above)
Comment (elton-furtado) [reply CHANGELOG/CHANGELOG-0.5.md]: done

**Review by jianjuns — COMMENTED** [review]
LGTM

**Review by salv-orlando — APPROVED** [review]
LGTM!

**Pull request merged**